statsmodels/graphics/gofplots.py

```diff
@@ -307,9 +357,21 @@ def qqplot(self, xlabel=None, ylabel=None, line=None, other=None,
             check_other = isinstance(other, ProbPlot)
             if not check_other:
                 other = ProbPlot(other)
-
-            fig, ax = _do_plot(other.sample_quantiles,
-                               self.sample_quantiles,
+            
+            s_self = self.sample_quantiles
+            s_other = other.sample_quantiles
+            
+            if len(s_self) > len(s_other):
+                raise ValueError("Sample size of `other` must be equal or " +
+                                 "larger than this `ProbPlot` instance")
+            elif len(s_self) < len(s_other):
+                # Use quantiles of the smaller set and interpolate quantiles of
+                # the larger data set
+                p = plotting_pos(self.nobs, self.a)
+                s_other = stats.mstats.mquantiles(s_other, p)
```

Comment:
is there anything specific to mstats.mquantiles, or can we use the more modern and possibly more efficient numpy functions, numpy.percentile.